### DISCUSSION TITLE
$31 redefine as laser/lathe mode,  vs. $32 being more appropriate?

### DISCUSSION THREAD

**Discussion by bLouChip** [grblHAL/core#253] [Q&A] [answered]
I was reading about the 'Additional and Extended Settings' in grblHAL recently and noticed the redefine of $31. 
See here: https://github.com/grblHAL/core/wiki/Additional-or-extended-settings
```
$31=<n> default value derived from DEFAULT_LASER_MODE and DEFAULT_LATHE_MODE.
Changed from original boolean for enabling laser mode.
0 - normal mode.  
1 - laser mode.  
2 - lathe mode.
```

$31 was not formerly (or in other grbl variants) a boolean for laser mode enable, that is $32.  $31 was the lower (min) value for PWM output signal.  I believe LightBurn relies on this definition, perhaps other CAM sw too. 

Shouldn't the redefine for normal vs. laser vs. lather be on the $32 config setting ? 
Sanity check please.

**Comment by terjeio** [comment, marked as answer]
Its a typo - should be $32, I'll fix it.
